tests/test_remote_client.py

```diff
@@ -0,0 +1,93 @@
+import os
+import re
+import time
+
+from importlib.util import find_spec
+
+import pytest
+import pexpect
+
+psutil = pytest.importorskip("psutil")
+
+pytestmark = pytest.mark.skipif(not find_spec("crossbar"),
+                              reason="crossbar required")
+
+def test_startup(crossbar):
+    pass
+
+@pytest.fixture(scope='function')
+def place(crossbar):
+    with pexpect.spawn('python -m labgrid.remote.client -p test create') as spawn:
+        spawn.expect(pexpect.EOF)
+        spawn.close()
+        assert spawn.exitstatus == 0, spawn.before.strip()
+
+    with pexpect.spawn('python -m labgrid.remote.client -p test set-tags board=bar') as spawn:
+        spawn.expect(pexpect.EOF)
+        spawn.close()
+        assert spawn.exitstatus == 0, spawn.before.strip()
+
+    yield
+
+    with pexpect.spawn('python -m labgrid.remote.client -p test delete') as spawn:
+        spawn.expect(pexpect.EOF)
+        spawn.close()
+        assert spawn.exitstatus == 0, spawn.before.strip()
+
+@pytest.fixture(scope='function')
+def place_acquire(place, exporter):
+    with pexpect.spawn('python -m labgrid.remote.client -p test add-match "*/Testport/*"') as spawn:
+        spawn.expect(pexpect.EOF)
+        spawn.close()
+        assert spawn.exitstatus == 0, spawn.before.strip()
+
+    with pexpect.spawn('python -m labgrid.remote.client -p test acquire') as spawn:
+        spawn.expect(pexpect.EOF)
+        spawn.close()
+        assert spawn.exitstatus == 0, spawn.before.strip()
+
+    yield
+
+    with pexpect.spawn('python -m labgrid.remote.client -p test release') as spawn:
+        spawn.expect(pexpect.EOF)
+        spawn.close()
+        assert spawn.exitstatus == 0, spawn.before.strip()
```

Comment:
This seems duplicated from `test_crossbar.py`. Please move them to `conftest.py` in a separate commit to allow them to be reused.